File: WebProjekat/WebProjekat/Models/RentacarGrade.cs
using WebProjekat.Requests.CarReservation;

namespace WebProjekat.Models
{
    public class RentacarGrade
    {
        public int Id { get; set; }
        public double Grade { get; set; }

        public virtual RentacarCompany Company { get; set; }
        public virtual User User { get; set; }

        public RentacarGrade() { }
        public RentacarGrade(GradeRentacarRequest gradeRentacarRequest)
        {
            Grade = gradeRentacarRequest.Grade;
        }
    }
}

File: ClientApp/src/app/models/rentacarGrade.ts
import { Car } from './car';
import { AuthUser } from './authUser';
import { RentacarCompany } from './rentacarCompany';

export class RentacarGrade {
    id: number;
    grade: number;
    company: RentacarCompany;
    user: AuthUser;
}